RELEASE_NOTES.md

```diff
@@ -129,10 +129,106 @@ include the new sigmoid module.  Options are now as follows:
+    increase the strength of the mask. This mode is now the default for
+    brush shapes.
+  
+  - It is now possible to change the set operator (mode) for all shapes
+    in a group from the right-click menu in the mask manager.
+  
+  - Some actions in the mask manager menu previously could be
+    activated even though they would have no impact on the image in some
+    contexts. The move up/down actions are therefore now disabled for
+    the first and last element in a group respectively, and it is no
+    longer possible to choose a set operator (mode) for the first
+    element in a group.
+  
+  - The sort order of the shapes in mask manager groups has been
+    reversed so that the lowest ranking shape is at the bottom of the
+    group. The sort order of shapes outside of a group has also been
+    changed for consistency.
+  
+  - It is now possible for all mask types to be added continuously in
+    the mask manager.
+
+  - Fixed an issue where the brush was not properly displayed after being
+    created from the mask manager, and a crash when creating gradients from the
+    mask manager has also been fixed.
+
+  - For all shapes the editable state is now properly set after creation
+     making it possible to move and resize the different parts of the shapes.
+  
+  - Fixed a display issue when editing a shape name in the mask manager.
+  
+  - Fixed the state of the set operators in the mask manager -- when
+    moving a mask up/down we now ensure that the first mask has no
+    operator and that the second one always has an operator assigned.
+    If no operator has been set yet the default (union) operator is
+    used.
+  
+  - When using a shortcut to add shapes to a drawn (blending) mask the
+    blending mode will automatically switch to "drawn mask" or "drawn &
+    parametric mask", depending on what it was before, so that any newly
+    created shape will actually affect the image.
+  
+  - In the drawn mask blending mode there used to be an "invert mask"
+    option that had the same functionality as the "toggle polarity" option.
+    Since both were doing the same thing, "invert mask" is now removed.
+  
+  - The drawing of masks has been reworked to ensure that all types of
+    masks are drawn in the same way -- the central area, border, and
+    highlighted segments are now displayed consistently. The highlighted
+    segment is now more visible, especially for the brush mask, where
+    the highlighted segment was barely distinguishable due to a bug.
+
+  - In the style and copy/paste dialogs a new column has been added to
+    show whether a module uses a mask (drawn/parametric/raster) using
+    the standard mask icon.
+  
+  - Editing of drawn masks has been improved -- it is now easier to
+    select masks' control points and path segments (in some cases it was
+    easy to accidentally select the whole path mask rather than a single
+    segment). On-canvas mask rendering has also been improved for better
```

Comment:
Is this different from "The drawing of masks has been reworked to ensure that all types of masks are drawn in the same way" above?

Reply:
No idea, I'm just the copy-editor, I don't understand half of what I'm editing

Reply:
@turbogit? Do you know/remember if these refer to different changes? (And if so, could you maybe clarify the difference?)

Reply:
@dterrahe : Yes different. The point you quote is about the display GUI only. The second point is about the way user's interact with the drawn masks.